app/src/localization/fr/index.ts

```diff
@@ -25,6 +25,11 @@ const translation = {
     "Warning": "Ensure only you have access to your wallet. (FR)",
     "UseToUnlock": "Use biometrics to unlock wallet? (FR)",
   },
+  "Credentials":{
```

Comment:
Able to add (FR) to these lines so we know where they come from in the app is in a different language?